tests/fixtures/animation.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <title>Share Button Test: Animation</title>
   <!-- temporary link tag -->
```

Comment:
we dont actually need the styles in the tests, it's more for a visual when using the chrome driver w/pioneer